### PR TITLE
Typical oruro food

### PR DESCRIPTION
**Pull request by ahuallpara**
#22

### CODE REVIEW

File: mains/Charquekan.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+### 3. Prepare the Mote:
+- If using pre-cooked mote, rinse it and heat it. If using dry mote, soak it overnight, then boil until tender.
+
+### 4. Fry the Onion and Chili:
+- In the same pan where you fried the charque, sauté the sliced onion and green chili (if using) with cumin, ground red chili, salt, and pepper until the onion softens.
+
+### 5. Assemble the Dish:
+- On a plate, place the boiled potatoes, a portion of mote, the crispy charque, and hard-boiled egg slices.
+- Garnish with the sautéed onion, parsley, and serve with a side of llajwa.
```
Comment (jprmz) [inline mains/Charquekan.md:33]:
```suggestion
- Garnish with the sautéed onion, parsley, and serve with a side of llajwa.

Enjoy!
```

File: mains/Charquekan.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- 2 hard-boiled eggs
+- 1 onion, sliced
+- 1-2 green chili peppers (optional)
+- 1 tablespoon ground red chili (ají)
+- 1 tablespoon chopped parsley
+- ½ teaspoon cumin
+- Oil for frying
+- Salt and pepper to taste
+- Llajwa (Bolivian spicy sauce) for serving
+
```
Comment (Gajera0703) [inline mains/Charquekan.md]:
```suggestion
## Istructions
```

**Review by Gajera0703 — CHANGES_REQUESTED** [review]

**Review by jprmz — CHANGES_REQUESTED** [review]
Add enjoy for a friendly ending

**Review by Camilogiu — CHANGES_REQUESTED** [review]
Please check the recipe again. I would have never thought you could have made such a huge mistake

File: mains/Charquekan.md [hunk 3]
```diff
@@ -14,7 +14,7 @@
 - Oil for frying
 - Salt and pepper to taste
 - Llajwa (Bolivian spicy sauce) for serving
-
+## Istructions
```
Comment (Camilogiu) [inline mains/Charquekan.md]: Please change to instructions, n is missing

**Review by Camilogiu — CHANGES_REQUESTED** [review]

**Pull request merged**